assignments/arrays.js

```diff
@@ -75,30 +324,59 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+console.log(`car ${inventory[32].id}.id is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = 49;
+console.log(inventory[lastCar].car_make + ' ' + inventory[lastCar].car_model);
```

Comment:
here you know how many items are in the array, but how would you solve this if you didn't (with for loops again)